doc/unison-manual.tex

```diff
@@ -1995,9 +1995,29 @@
 monitoring), interrupting with ``Ctrl-C'' or with signal \verb|SIGINT| or
 \verb|SIGTERM| works the same way as described above and will additionally stop
 the continuous process. To stop only the continuous process and let the last
-synchronization complete normally, send signal \verb|SIGUSR2| or \verb|SIGQUIT|
-(the latter can also be generated from keyboard by pressing
-``Ctrl-\textbackslash'' or ``Ctrl-4''), instead.
+synchronization complete normally, press ``Ctrl-D'' or send signal
+\verb|SIGUSR2| or \verb|SIGQUIT| (the latter can also be generated from
+keyboard by pressing ``Ctrl-\textbackslash'' or ``Ctrl-4''), instead.
+``Ctrl-D'' is expected to send an EOF if the sandard input is a terminal.
+Closing the input and receiving an EOF, a \verb|^C| (0x03), or a \verb|^D|
+(0x04) from the input terminal or a redirected standard input have the same
+effect. If the standard input is redirected from a regular file, not open or
+not open for reading already at Unison startup then it is ignored and only
+signals can be used to send the stop request.
+
+\begin{quote}
+  {\em Tips:} For continuous synchronization the input should not be
+  redirected from \verb|/dev/null| (as that would stop the process after
+  the first synchronization). This is what some implementations of \verb|nohup|
+  may do if run from a terminal without redirecting input. Input should
+  neither be redirected from any source providing a lot of input (such as
+  a device or a pipe that produces large quantities of data). In such cases,
+  Unison would keep reading all the input looking for the stop condition.
```

Comment:
I think it's reasonable for someone to run a process < /dev/null (and 2>&1 > /dev/null), and saying this isn't going to work seems like a POLA violation.

Reply:
Fair enough, I can remove EOF from being a "stop condition" for anything other than interactive terminal. (It's already removed for regular files.)